documentation/content/docs/ansible.md

```diff
@@ -146,10 +146,18 @@ $ ansible-playbook                                              \
    -u ubuntu                                                    \
    -e hostname=ec2-35-163-16-21.us-west-2.compute.amazonaws.com \
    -e gpu=yes                                                   \
-   -e install_nvidia_driver=yes                                 \
+   -e install_nvidia_driver=no                                  \
    --private-key=./paraview-52.pem
 ```
 
+Since the ansible script won't install the drivers for the GPU you will have to do it yourself following the instruction from [NVidia](http://www.nvidia.com/Download/index.aspx?lang=en-us)
+
+The driver we used at the time of this documentation was:
+
+<center>
```

Comment:
Beware, [\<center\>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/center) is a deprecated web tag.